test/e2e/validation_test.go

```diff
@@ -46,6 +46,39 @@ type winService struct {
 	description string
 }
 
+// testKubeletPriority tests if kubelet priority class is set to "AboveNormal"
+func testKubeletPriority(t *testing.T) {
+	tc, err := NewTestContext()
+	require.NoError(t, err)
+	requiredPriorityClass := "AboveNormal"
+
+	if len(gc.allNodes()) == 0 {
+		t.Skip("0 expected nodes")
+	}
```

Comment:
Should we skip in this case? Elsewhere in the test suite we `require` > 0 nodes
e.g. 
- require.Greater(t, len(gc.allNodes()), 0, "test requires at least one Windows node to run")
- require.NotEmpty(t, gc.allNodes())